src/main/java/org/sagebionetworks/web/client/view/ProfileViewImpl.java

```diff
@@ -516,22 +545,59 @@ public void setTabSelected(Synapse.ProfileArea targetTab) {
 
     if (targetTab == Synapse.ProfileArea.PROFILE) {
       pageHeaderTitle.setText(profileHeaderText);
+      orientationBannerPanel.clear();
       DisplayUtils.show(profileTabContainer);
     } else if (targetTab == Synapse.ProfileArea.PROJECTS) {
       pageHeaderTitle.setText("Your Projects");
+      configureOrientationBanner(
+        "Projects",
+        "Getting Started With Your Projects",
+        "Projects are the main \"containers\" where information is stored and organized in Synapse. They are online workspaces where you can collaborate and share your work with teammates. Projects can be shared with individuals, small teams, or large consortia. Projects can be private so only you and your team can see what\'s inside, or they can be shared publicly for anyone to browse.",
+        null,
+        null,
+        "Learn More About Projects",
+        "https://help.synapse.org/docs/Setting-Up-a-Project.2055471258.html"
+      );
       DisplayUtils.show(projectsTabContainer);
     } else if (targetTab == Synapse.ProfileArea.TEAMS) {
       pageHeaderTitle.setText("Your Teams");
+      configureOrientationBanner(
+        "Teams",
+        "Getting Started With Your Teams",
+        "Teams allow you to easily manage groups of users to control access to projects, communicate with colleagues, and participate in challenges.",
+        "Search Teams",
+        "https://www.synapse.org/#!TeamSearch:",
+        "Learn More About Teams",
+        "https://help.synapse.org/docs/Teams.1985446029.html"
+      );
       DisplayUtils.show(teamsTabContainer);
     } else if (targetTab == Synapse.ProfileArea.SETTINGS) {
       pageHeaderTitle.setText("Account Settings");
+      orientationBannerPanel.clear();
       DisplayUtils.show(settingsTabContainer);
     } else if (targetTab == Synapse.ProfileArea.CHALLENGES) {
       pageHeaderTitle.setText("Your Challenges");
+      configureOrientationBanner(
+        "Challenges",
+        "Getting Started With Your Challenges",
+        "Challenges are open science, collaborative competitions for evaluating and comparing computational algorithms or solutions to problems.",
+        null,
+        null,
+        "Learn More About Challenges",
+        "https://help.synapse.org/docs/Challenges.1985184148.html"
+      );
       DisplayUtils.show(challengesTabContainer);
     } else if (targetTab == Synapse.ProfileArea.FAVORITES) {
       pageHeaderTitle.setText("Your Favorites");
-
+      configureOrientationBanner(
+        "Favorites",
+        "Getting Started With Your Favorites",
+        "Throughout Synapse, you can favorite pretty much any item (project, file, folder, table, dataset, etc.) by clicking the star icon next to its name. This will add that item to your favorites list.",
+        null,
+        null,
+        "Learn More About Favorites",
+        "https://help.synapse.org/docs/Navigating-Synapse.2048557182.html#NavigatingSynapse-Favorites"
```

Comment:
I found a docs page related to Favorites, so included it here. However, [the design](https://www.figma.com/file/bALXz8AOxdzQyrsSd4RkVm/Governance-UI%3A-Registration-and-Data-Access?type=design&node-id=2557-12762&t=XtpKoMaYnPxvytt6-4) doesn't include a docs link for Favorites, so happy to remove the link if that's preferred!

Reply:
Good call!